app/views/fae/pages/home.html.slim

```diff
@@ -1,25 +1,53 @@
 header.content-header.js-content-header.-dashboard
-  i.icon-home>
-  | &nbsp;&nbsp;Welcome,
-  strong< = current_user.full_name
+ i.icon-home>
+ | &nbsp;&nbsp;Welcome,
+ strong< = current_user.full_name
 
 == render 'flash_messages'
 
 main.content
-  table.js-sort-column
-    thead
-      tr
-        th Name
-        th Type
-        th.-action-wide Modified
-    tbody
-      - @list.each do |item|
-        - begin
-          - parent = item.respond_to?(:fae_parent) ? item.fae_parent : nil
-          - edit_path = edit_polymorphic_path([main_app, fae_scope, parent, item])
-          - index_path = polymorphic_path([main_app, fae_scope, parent, item.class.to_s.pluralize.underscore])
-          tr
-            td: a href=edit_path = item.fae_display_field
-            td: a href=index_path = item.class.to_s
-            td = fae_date_format item.updated_at
-        - rescue
+- if @list.empty?
```

Comment:
I'll be the first to admin the dashboard is severely lacking. I like the concept, however we have two separate audiences with Fae: the developers who build with Fae and the clients who use Fae to manage our content. If our non-tech savvy clients jump in and they haven't created any objects, this message will appear and likely confuse them.

I could see an argument for the message if no models have been created yet, as the CMS wouldn't be client ready at that point. You could apply the same conditional, but tap into `all_models` defined in `Fae::ApplicationController`.

Reply:
Glad to hear that this could be a useful addition to the Fae interface! The non-tech savvy audience receiving this message definitely presents an opportunity for confusion. I'll take a look at `all_models`.